terraform/main.tf

```diff
@@ -0,0 +1,115 @@
+terraform {
+  required_version = "~> 1.3"
+
+  required_providers {
+    digitalocean = {
+      source  = "digitalocean/digitalocean"
+      version = "~> 2.0"
+    }
+  }
+}
+
+provider "digitalocean" {
+  token = var.digitalocean_token
+}
+
+# resource "digitalocean_ssh_key" "new-key" {
+#   name       = "new-key"
+#   public_key = file("/Users/mac/.ssh/id_rsa.pub")
+# }
+
+# resource "digitalocean_spaces_bucket" "spaces-name" {
+#   name   = "spaces-name"
+#   region = "nyc3"
+# }
+
+resource "digitalocean_volume" "forest-volum" {
```

Comment:
It should be rather `forest-volume`, no? Or better, something parameterizable.

Reply:
Sure